### ISSUE TITLE
Feature toggle project.json error

### ISSUE THREAD

**Issue by johnkors** [OmniSharp/omnisharp-vscode#1920]
There's no feature toggle to disable the pesky warning about me having a project.json file in a folder. Could that be added? I'm fully aware I'm behind on updating certain projects to the new SDK style projects, and that they are obsolete. As it is now I need to click a warning pop-up for every folder containing a project.json. 

Would appreciate it if I could disable this seatbelt. :)

## Steps to reproduce
Open a folder with a project.json file.

## Expected  behavior
If feature toggled off, nothing.

## Actual behavior
A warning.
![image](https://user-images.githubusercontent.com/206726/33894364-17dade76-df5e-11e7-9822-53ccd58edb49.png)

**Comment by DustinCampbell** [comment]
We don't have plans to remove this warning. Eventually, project.json will likely be dropped completely, so we highly encourage you to migrate any projects that you wish to continue using with the latest version of C# for VS Code to .csproj.

**Comment by johnkors** [comment]
Yep, as mentioned - I'm fully aware of all those statements. That's what's _so_ annoying with this pop-up. Even though I'm fully aware of my projects being behind on updates AND have a migration plan, I still get a re-occuring warning about it. 

Today I need to click away pop-ups like a ad-fested  internet page anno 2001. Could it at least be moved to away from a pop-up (which I cannot close without the use of a mouse), and into the console..? 

These pop-ups are very intrusive from a usability point of view.

**Comment by DustinCampbell** [comment]
Can you not close the pop up with the Escape key?

**Comment by DustinCampbell** [comment]
cc @Andrew-MSFT

**Comment by johnkors** [comment]
Yeah, ESC works as a close dialogue as well as the mouse.

**Comment by johnkors** [comment]
There's an equivalent setting for restore, `"csharp.suppressDotnetRestoreNotification"` . The newest SDK no longer requires a restore, but CLI tools still do need one. So similar to this setting, I think it makes sense to provide the option for the developer to customize as one sees fit. But maybe it's just me.

**Comment by DustinCampbell** [comment]
Fair enough. I can see the act of enabling a setting to disable the warning is an implicit acknowledgement that the feature is going out of support. Just note that the project.json support will likely be removed completely at some point in the future.

**Comment by rchande** [comment]
@akshita31 Do you want to take a stab at this? You'll want to define a new option type in https://github.com/OmniSharp/omnisharp-vscode/blob/master/package.json . You will need to find the code that issues this warning and update it to look at the option. Here's an example of how to look at options: https://github.com/OmniSharp/omnisharp-vscode/blob/master/src/features/codeLensProvider.ts#L56. It would also be great if we could have an integration test for this scenario. Thanks!

**Comment by akshita31** [comment]
This issue has been fixed in the 1.14 milestone. We just released a beta of 1.14 - https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4. Follow [these instructions](https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4) to install a beta release of C# for VS Code.
